topics/community/tutorials/community_content/tutorial.md

```diff
@@ -0,0 +1,164 @@
+---
+layout: tutorial_hands_on
+priority: 1
+subtopic: sig
+title: "Creating community content"
+questions:
+  - "What content does my Topic automatically get from the GTN?"
+  - "How can I make sure this data is collected well?"
+objectives:
+  - "Signpost community leads and users to useful resources"
+  - "Explain why metadata is key for such community resources"
+  - "Provide a reference, rather than a tutorial"
+time_estimation: "30"
+key_points:
+  - "The GTN has worked hard to provide automated metrics and resources to highlight and acknowledge the efforts of communities and community leads"
+  - "This only works if we contribute with effective tagging."
+requirements:
+contributions:
+  authorship:
+  - nomadscientist
+  requirements:
+  -
+      type: "internal"
+      topic_name: community
+      tutorials:
+          - sig_define
+---
+
+Galaxy *[Special Interest Group](https://galaxyproject.org/community/sig)* (**SIG**)s work hard to build and maintain training resources. The GTN has worked hard to acknowledge this and offer nice impact pages to communities.
+
+Here is a list of resources that you can use!
+
+> <comment-title></comment-title>
+> - We want this material to grow (similar to the 'Creating content in markdown' tutorial) so please do add further resources that can help communities!
+{: .comment}
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Maintainer page
+
+Maintaining resources is **fundamental** to the quality and usefulness of any software resource. We do *not* throw code over the wall!
+
+To help topic maintainers to quickly recognise what materials need updating and fixing, **Topic Maintainer Pages** were built.
+
+> <hands-on-title>Go to your topic maintainer page</hands-on-title>
+>
+> 1. Go to any training topic of interest. This will have the form
+>    ```
+>     https://training.galaxyproject.org/training-material/topics/<your topic here>
```

Comment:
https://training.galaxyproject.org/training-material/topics/admin/#community they're also linked at the bottom

Reply:
Holy shit, that's epic, never noticed

Reply:
added an FAQ